src/styles/core-image.scss

```diff
@@ -84,3 +84,15 @@
 		}
 	}
 }
+
+.wp-block-image.is-style-report-image.alignfull figcaption {
+	max-width: #{ variables.$width-breakpoint-wide + 32px };
+	margin-inline: auto;
+}
+
+// Remove padding for group wrappers around report images.
+.wp-block-group.has-background:has(
+	.wp-block-image.is-style-report-image:only-child,
+	.wp-block-image.is-style-report-image-vertical:only-child ) {
```

Comment:
```suggestion
	> .wp-block-image.is-style-report-image:only-child,
	> .wp-block-image.is-style-report-image-vertical:only-child ) {
```
My only question with this, is whether we should scope the child block checks to direct descendants. I could imagine a structure like this:

```
.wp-block-group.has-background
    .wp-block-heading
    .wp-block-group
        .wp-block-image.is-style-report-image
```
This is _probably_ excessively hypothetical, since the report image style variation itself is being used. But There's no actual guard against nesting it in another group, and this could therefore affect a parent group.

Reply:
Wise 👍 Good catch. 

I amended this slightly, since the actual markup has an inner container div we need to account for.

<img width="656" alt="Screenshot 2025-03-27 at 21 50 59" src="https://github.com/user-attachments/assets/0793c4c4-9fa7-4515-ab70-fa217f726332" />
<img width="1065" alt="Screenshot 2025-03-27 at 21 50 19" src="https://github.com/user-attachments/assets/61b9b93b-a1ce-4f82-bf8f-a24de6a0e344" />